docs/_articles/en/scanner-commands/run.md

```diff
@@ -72,7 +74,7 @@ sfdx scanner:run -t <array> [-c <array>] [-r <array>] [-e <array>] [-f
 
 ## Additional Notes
 
-- `--ruleset` option is deprecated and will be removed soon. Please use --category instead.
+- `--ruleset` option is deprecated. Use --category instead.
 - `-v/--violations-cause-error` flag is deprecated. Please use `-s/--severity-threshold` instead. 
```

Comment:
If we removed the "Please" in line 77, should we remove it in line 78 too?

Reply:
Yes, thanks @jfeingold35 . Removing please.